paraflow-loader-consumer/src/main/java/cn/edu/ruc/iir/paraflow/loader/consumer/threads/DataFlushThread.java

```diff
@@ -64,6 +70,27 @@ public void run()
 
     private void flushMeta(BufferSegment segment)
     {
-        System.out.println(segment.getFilePath());
+        String topic = segment.getFiberPartitions().get(0).getTopic();
+        int indexOfDot = topic.indexOf(".");
+        String dbName = topic.substring(0, indexOfDot);
+        int length = topic.length();
+        String tblName = topic.substring(indexOfDot + 1, length);
+        int fiberValue;
+        String path = segment.getFilePath();
+        long beginTime = segment.getTimestamps()[0];
+        long endTime = segment.getTimestamps()[0];
+        for (long timeStamp : segment.getTimestamps()) {
+            if (timeStamp < beginTime) {
+                beginTime = timeStamp;
+            }
+            if (timeStamp > endTime) {
+                beginTime = timeStamp;
```

Comment:
shouldn't this be `endTime = timestamp` ??

Reply:
No need to calculate segment level begin time and end time.
More are explained later.